src/MeetingsSDKAdapter.js

```diff
@@ -535,6 +767,30 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
         map((event) => this.attachMedia(ID, event))
       );
 
+      const meetingWithMediaStoppedSharingLocalEvent$ = fromEvent(sdkMeeting, EVENT_MEETING_STOPPED_SHARING_LOCAL).pipe(
+        // eslint-disable-next-line no-console
+        map(() => console.log('EVENT_MEETING_STOPPED_SHARING_LOCAL is triggered', this)),
```

Comment:
Can you elaborate on what `meetingWithMediaStoppedSharingLocalEvent$` and `meetingWithMediaStartedSharingLocalEvent$` are trying to accomplish?
I feel that this is misplaced.

Reply:
These are similar to meetingWithMediaStoppedEvent$ in getMeeting().  

meetingWithMediaStartedSharingLocalEvent$ is the event when local sharing stream is started and added to SDP offer.
meetingWithMediaStoppedSharingLocalEvent$ is the event monitoring the ```onended``` event of the local sharing stream. If the user clicks the "stop share" button (which is native button of browser), this event will help to detect that the user wants to stop share.
Both events take effects during the whole meeting period, just like meetingWithMediaStoppedEvent$.

<img width="1674" alt="Screen Shot 2020-06-03 at 11 57 33 AM" src="https://user-images.githubusercontent.com/22400766/83677731-6e719580-a591-11ea-9fc0-aef495f88822.png">


Maybe a better name for this event to increase the readability?

Reply:
I think the logic here is not right. Whatever happens in `getMeeting` should result in the final observable emitting a `Meeting` object as defined by the component system data model. The resulting emission should have the desired state.
For instance, if a new local share stream is added, then the `Meeting` object should have that stream. If the streamed is stopped, then it should be removed, etc.
The actions are invoked by the component when the user clicks the control and should not be invoked directly in the adapter.